native/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -53,14 +50,15 @@ public class Init implements BLauncherCmd {
     private final PrintStream outStream = System.out;
     private final PrintStream errStream = System.err;
 
+    private static Path configPath = Paths.get("config.toml");
```

Comment:
It needs to be changed as `Config.toml` and made a constant.

Reply:
updated in [dbd5790](https://github.com/ballerina-platform/persist-tools/pull/5/commits/dbd57908d8eee997f82b75e59978f916f25b75f2)